cypress/TestCases/FireboltCertification/UserGrants.md

```diff
@@ -0,0 +1,87 @@
+# Firebolt Certification
+Feature file for Firebolt Certification UserGrants Module.
+
+## Purpose
+A module for managing grants given by the user.
+
+## Requirements :
```

Comment:
The readme is not proper. How do you know if below is true for all platforms? 
* The corresponding grant policy of the capability being tested must be added inside the "grantPolicies" object within the "capabilities" object.

README must be written in such a way that is useful for folks who wants to test

Reply:
Updated structure of grantploicy